packages/desktop-client/src/components/future/index.tsx

```
@@ -0,0 +1,1405 @@
+                                        fontWeight: '500',
+                                      }}
+                                    >
+                                      {formatCurrency(category.projectedAmount)}
+                                    </Text>
+                                  )}
+                                </View>
+                              ))}
+                          </View>
+                        </View>
+                      ))}
+                  </View>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)